src/GoogleFit/FitOnFhir.GoogleFit/Services/UsersService.cs

```diff
@@ -111,5 +119,41 @@ public override async Task QueueFitnessImport(User user, CancellationToken cance
                 await _queueService.SendQueueMessage(user.Id, googleUserInfo.UserId, googleUserInfo.PlatformName, cancellationToken);
             }
         }
+
+        public override async Task<bool> RevokeAccess(string patientId, string system, CancellationToken cancellationToken)
```

Comment:
I actually don't think you need to return a bool here. The reason is, if a Patient, User, or UserPlatformInfo is not found, we should just return a 200. If we don't, the caller of the API needs to handle the error, in which case will probably be ignored because revoking access to something that doesn't have access is not really an error.